python/ray/tune/stopper.py

```diff
@@ -43,6 +46,27 @@ def stop_all(self):
 
 
 class CombinedStopper(Stopper):
+    """Combine several stoppers.
```

Comment:
Is it worth mentioning that this combines Stoppers via OR and not AND, i.e stopping conditions are determined based on if any of the stopping conditions are met rather than all?

Reply:
Hmm, this is internal API now (not publicly listed on the documentation), so I'm guessing users will need to read source code anyways to read this. I'll add a small note though.